### DISCUSSION TITLE
Auto-AI-tagging, is it possible to correct?

### DISCUSSION THREAD

**Discussion by tomasenskede** [photoprism/photoprism#4482] [Q&A]
![image](https://github.com/user-attachments/assets/174cea39-d450-43cd-8427-344913fe5682)

Auto-tagging is cool but sometimes its incorrect... is it possible to correct then?

**Comment by graciousgrey** [comment]
You can remove labels that are not a good fit or add new ones:
- https://docs.photoprism.app/user-guide/organize/labels/#removedelete-labels-from-photos
- https://docs.photoprism.app/user-guide/organize/labels/#add-labels-to-photos
